pyglmnet/pyglmnet.py

```diff
@@ -65,6 +65,13 @@ class GLM(object):
         where lambda is number in reg_lambda list
         default: None, a list of 10 floats spaced logarithmically (base e)
         between 0.5 and 0.01 is generated.
+    group: list or ndarray
+        dimension: n_features
+        each entry of the list/ array should contain an int from 1 to n_groups
+        that specify group membership for each parameter (not beta_0)
+        note: if you do not want to specify a group for a specific parameter,
+        set it to zero
+        default: None
```

Comment:
Typically, `None` is added where you list the data types because `None` is neither a `list` nor an `ndarray`. See:
```py
>>> type(None)
```
Something like:

`group: list | ndarray | None`